### ISSUE TITLE
推理耗时

### ISSUE THREAD

**Issue by Mr-Jin2** [siyuhuang/QuantArt#7]
RTX4090 推理4张图片需要十几分钟，是否正常？
-----------------------------------------------------------------
(quantart)% bash test.sh
Running on GPUs 0,
Working with z of shape (1, 256, 16, 16) = 65536 dimensions.
VQLPIPSWithDiscriminator running with hinge loss.
Restored from logs/landscape2art/checkpoints/last.ckpt
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
accumulate_grad_batches = 1
Setting learning rate to 4.50e-06 = 1 (accumulate_grad_batches) * 1 (num_gpus) * 1 (batchsize) * 4.50e-06 (base_lr)
/home/kas/.conda/envs/quantart/lib/python3.8/site-packages/torch/cuda/__init__.py:104: UserWarning: 
NVIDIA GeForce RTX 4090 with CUDA capability sm_89 is not compatible with the current PyTorch installation.
The current PyTorch install supports CUDA capabilities sm_37 sm_50 sm_60 sm_61 sm_70 sm_75 compute_37.
If you want to use the NVIDIA GeForce RTX 4090 GPU with PyTorch, please check the instructions at https://pytorch.org/get-started/locally/

  warnings.warn(incompatible_device_warn.format(device_name, capability, " ".join(arch_list), device_name))

Testing 0 / 4
Testing 1 / 4
Testing 2 / 4
Testing 3 / 4

(quantart)% tree /home/kas/code/QuantArt/datasets
/home/kas/code/QuantArt/datasets
├── all_data_info.csv.zip
├── lhq_1024_jpg
│   └── lhq_1024_jpg
│       ├── 1.jpg
│       ├── 2.jpg
│       ├── 3.jpg
│       └── 4.jpg
├── painter-by-numbers
│   ├── test
│   │   ├── 1.jpg
│   │   ├── 2.jpg
│   │   ├── 3.jpg
│   │   └── 4.jpg
│   └── train_info.csv
└── train_info.csv

(quantart) % tree logs 
logs
└── landscape2art
    ├── checkpoints
    │   └── last.ckpt
    └── configs
        ├── test_paired.yaml
        └── test.yaml

**Comment by siyuhuang** [comment]
Seems like your PyTorch version is not compatible with RTX4090. Please try to update your PyTorch version.
